site/layouts/partials/head-docs.html

```diff
@@ -8,4 +8,5 @@
   {{ $styles := resources.Get "styles.scss" | toCSS $options | resources.Fingerprint }}
   <link rel="stylesheet" href="{{ $styles.RelPermalink }}" integrity="{{ $styles.Data.Integrity }}">
   {{/* TODO  {% seo %}*/}}
```

Comment:
```suggestion
  {{/* TODO  {% seo %}*/}}
  <link rel="preconnect" href="https://9ASKQJ1HR3-dsn.algolia.net" crossorigin />
```
per https://docsearch.algolia.com/docs/docsearch-v3/#preconnect